docs/en/workshop/a-molecular-unfolding/permissions.json

```diff
@@ -94,10 +92,10 @@
             ],
             "Resource": [
                 "arn:aws:s3:::qc*/*",
-                "arn:aws:s3:::qr*/*",
                 "arn:aws:s3:::braket*/*",
```

Comment:
the bucket of service team?

Reply:
if you enable braket service, it will create bucket braket-xxxxx in your account, so make sure we can access it.